libretro/libretro-core.c

```diff
@@ -1968,15 +2158,36 @@ void retro_set_environment(retro_environment_t cb)
          {{ NULL, NULL }},
          "RETROK_END"
       },
+        
+//  Video Cropping Hot Keys Menu Items
 #if defined(__X64__) || defined(__X64SC__) || defined(__X128__) || defined(__VIC20__) || defined(__PLUS4__)
       {
          "vice_mapper_zoom_mode_toggle",
-         "Hotkey: Toggle Zoom Mode",
-         "Press the mapped key to toggle zoom mode.",
+         "Hotkey: Toggle Zoom Mode (Horizontal + Vertical Cropping)",
+         "Press the mapped key to toggle Zoom ON/OFF.",
          {{ NULL, NULL }},
          "---"
       },
+
+      {
+         "vice_mapper_bcrop_horiz_mode_cycle",

```

Comment:
Does this hotkey choice actually change the option in quickmenu->options? For example, if the user uses the hotkey, then opens and closes quickmenu, does that reset the option or does it actually stay?

Reply:
By design, like the original Zoom Hot Key Tony implemented, it does NOT alter the menu setting. That would not be the correct thing to do as Core Options are saved by RA and pressing some hot-keys during a game shouldn't undo what you've already previously saved.